packages/ui-react/src/components/Payment/Payment.tsx

```diff
@@ -221,19 +220,12 @@ const Payment = ({
               ) : (
                 showChangeSubscriptionButton && (
                   <Button
-                    className={styles.upgradeSubscription}
-                    label={t('user:payment.change_subscription')}
+                    className={styles.changePlanCancelButton}
+                    label={t('user:payment.cancel_subscription')}
+                    onClick={onCancelSubscriptionClick}
+                    variant="danger"
                     disabled={!canRenewSubscription && activeSubscription.status === 'cancelled'}
-                    onClick={() => {
-                      if (offers.length > 1 && !canRenewSubscription) {
-                        setIsChangingOffer(true);
-                      } else {
-                        onUpgradeSubscriptionClick?.();
```

Comment:
We can't remove this part, it is part of the Cleeng flow.

Reply:
can you check it again ?